Exiled.Events/Patches/Events/Player/ChangingRole.cs

```diff
@@ -193,16 +178,20 @@ private static void UpdatePlayerRole(RoleTypeId newRole, API.Features.Player pla
             player.MaxHealth = default;
         }
 
-        private static void ChangeInventory(API.Features.Player player, List<ItemType> items, Dictionary<ItemType, ushort> ammo, RoleTypeId prevRole, RoleTypeId newRole, RoleChangeReason reason)
+        private static void ChangeInventory(ChangingRoleEventArgs ev, RoleTypeId prevRole)
         {
+            API.Features.Player player = ev.Player;
+            List<ItemType> items = ev.Items;
+            Dictionary<ItemType, ushort> ammo = ev.Ammo;
+            RoleChangeReason reason = (RoleChangeReason)ev.Reason;
+            RoleTypeId newRole = ev.NewRole;
```

Comment:
You should just use `ev.X` below instead of making new vars.

Reply:
I was trying to be quick but sure